account_move_reconcile_helper/README.rst

```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+account_move_reconcile_helper
+=============================
+
+This module was written to offer facilities about reconciliation
```

Comment:
Sorry, I only see the model part. Anyways, please add that also include a field for the balance of the line (credit_debit_balance)